THIRDPARTY.md

```diff
@@ -0,0 +1,24 @@
+This project uses icons by [Material Design Icons](https://materialdesignicons.com/).
+
+## Pictogrammers Free License
+
+This icon collection is released as free, open source, and GPL friendly by
+the [Pictogrammers](http://pictogrammers.com/) icon group. You may use it
+for commercial projects, open source projects, or anything really.
+
+# Icons: Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
We should be distributing the license with our software here, I don't think just linking to it cuts it legally. Let's embed Apache 2.0 in here, akin to how THIRD-PARTY-LICENSES.txt works over on green.

Reply:
Okay your call, but it's a big boy...